frontend/util/types/fhir.ts

```diff
@@ -1,3 +1,17 @@
 // type representing the resource types that are relevant to the Measure Repository Service
 export type FhirArtifact = fhir4.Measure | fhir4.Library;
 export type ArtifactResourceType = FhirArtifact['resourceType'];
+
+/**
+ * Interface containing fields pulled from the resource JSON that are used
+ * to populate the Resource Info Card.
+ */
+export interface ResourceInfo {
```

Comment:
Overall I like the approach of specifying what info we'll render on the actual page to limit the amount of JSON being sent as props (i.e. instead of sending over the entire list of resources or search set bundle)